ts/Series/SolidGauge/SolidGaugeSeriesDefaults.ts

```diff
@@ -41,6 +41,20 @@ import type SolidGaugeSeriesOptions from './SolidGaugeSeriesOptions';
  */
 const SolidGaugeSeriesDefaults: SolidGaugeSeriesOptions = {
 
+    /**
+     * The corner radius of the border surrounding solid gauge. A
+     * number signifies pixels and percentage string, like for example `50%`,
```

Comment:
As a good practice, it's best to fill in the whole available space of 80 characters and not leave vowels, conjuctions or hypens at the end of the line.
```suggestion
     * The corner radius of the border surrounding solid gauge. A number
     * signifies pixels and percentage string, like for example `50%`,
```